### DISCUSSION TITLE
Duplicate role on role assign

### DISCUSSION THREAD

**Discussion by Kry9toN** [Budibase/budibase#11630] [Help] [answered]
hei there, someone can help me to fix this issue

![image](https://github.com/Budibase/budibase/assets/44697929/a3727cad-b05b-40b3-b61e-11ec5020433b)

**Comment by tglynx** [comment]
Hi,
wow, have you tried to clear the browser cache? This is a self hosted environment i suppose?

**Reply by Kry9toN** [reply]
i already clear cache in my browser and still like that, yes in self host but not use local network

**Reply by tglynx** [reply]
see below, the issue has been reproduced and a bug has been raised for this issue!

**Comment by ConorWebb96** [comment]
Are you able to reproduce this on other apps? Can you create a bug report with replication steps here; https://github.com/Budibase/budibase/issues/new/choose

**Reply by melohagan** [reply, marked as answer]
I have raised a bug here with the reproduction steps: https://github.com/Budibase/budibase/issues/11666

It's worth noting this is a visual thing, and will not affect the functioning of the rbac within your apps.
